csharp/private/providers.bzl

```diff
@@ -23,82 +23,80 @@ def _make_csharp_provider(tfm):
-CSharpAssembly_netstandard13 = _make_csharp_provider("netstandard1.3")  # buildifier: disable=name-conventions
-CSharpAssembly_netstandard14 = _make_csharp_provider("netstandard1.4")  # buildifier: disable=name-conventions
-CSharpAssembly_netstandard15 = _make_csharp_provider("netstandard1.5")  # buildifier: disable=name-conventions
-CSharpAssembly_netstandard16 = _make_csharp_provider("netstandard1.6")  # buildifier: disable=name-conventions
-CSharpAssembly_netstandard20 = _make_csharp_provider("netstandard2.0")  # buildifier: disable=name-conventions
-CSharpAssembly_netstandard21 = _make_csharp_provider("netstandard2.1")  # buildifier: disable=name-conventions
-CSharpAssembly_net11 = _make_csharp_provider("net11")  # buildifier: disable=name-conventions
-CSharpAssembly_net20 = _make_csharp_provider("net20")  # buildifier: disable=name-conventions
-CSharpAssembly_net30 = _make_csharp_provider("net30")  # buildifier: disable=name-conventions
-CSharpAssembly_net35 = _make_csharp_provider("net35")  # buildifier: disable=name-conventions
-CSharpAssembly_net40 = _make_csharp_provider("net40")  # buildifier: disable=name-conventions
-CSharpAssembly_net403 = _make_csharp_provider("net403")  # buildifier: disable=name-conventions
-CSharpAssembly_net45 = _make_csharp_provider("net45")  # buildifier: disable=name-conventions
-CSharpAssembly_net451 = _make_csharp_provider("net451")  # buildifier: disable=name-conventions
-CSharpAssembly_net452 = _make_csharp_provider("net452")  # buildifier: disable=name-conventions
-CSharpAssembly_net46 = _make_csharp_provider("net46")  # buildifier: disable=name-conventions
-CSharpAssembly_net461 = _make_csharp_provider("net461")  # buildifier: disable=name-conventions
-CSharpAssembly_net462 = _make_csharp_provider("net462")  # buildifier: disable=name-conventions
-CSharpAssembly_net47 = _make_csharp_provider("net47")  # buildifier: disable=name-conventions
-CSharpAssembly_net471 = _make_csharp_provider("net471")  # buildifier: disable=name-conventions
-CSharpAssembly_net472 = _make_csharp_provider("net472")  # buildifier: disable=name-conventions
-CSharpAssembly_net48 = _make_csharp_provider("net48")  # buildifier: disable=name-conventions
-CSharpAssembly_netcoreapp10 = _make_csharp_provider("netcoreapp1.0")  # buildifier: disable=name-conventions
-CSharpAssembly_netcoreapp11 = _make_csharp_provider("netcoreapp1.1")  # buildifier: disable=name-conventions
-CSharpAssembly_netcoreapp20 = _make_csharp_provider("netcoreapp2.0")  # buildifier: disable=name-conventions
-CSharpAssembly_netcoreapp21 = _make_csharp_provider("netcoreapp2.1")  # buildifier: disable=name-conventions
-CSharpAssembly_netcoreapp22 = _make_csharp_provider("netcoreapp2.2")  # buildifier: disable=name-conventions
-CSharpAssembly_netcoreapp30 = _make_csharp_provider("netcoreapp3.0")  # buildifier: disable=name-conventions
+CSharpNetStandardAssemblyInfo = _make_csharp_provider("netstandard")
```

Comment:
Ah, I got confused for a bit. You probably want something other than `AssemblyInfo` because that's already a thing in .NET. Does it have to end in `Info`? Could it just be `CSharpAssemblyNet40` etc.?

Reply:
It does need an `Info` suffix. Maybe `CSharpAssemblyNet20Info`?

Reply:
I have change the naming format from `CSharp<Framework>AssemblyInfo` to `CSharpAssembly<Framework>Info`